content/docs/reference-react-component.md

```diff
@@ -143,47 +143,47 @@ If you need to interact with the browser, perform your work in `componentDidMoun
 
-The constructor for a React component is called before it is mounted. When implementing the constructor for a `React.Component` subclass, you should call `super(props)` before any other statement. Otherwise, `this.props` will be undefined in the constructor, which can lead to bugs.
+Egy React komponense konstruktora a DOM-ba helyeződés előtt hívódik meg. Egy `React.Component` konstruktorában először a `super(props)`-t kell meghívni minden más hívás előtt. Enélkül `this.props` undefined értékű lesz a konstruktorban, ami hibákhoz vezethet.
 
-Typically, in React constructors are only used for two purposes:
+A React konstruktorokat általában kétféle okból használjuk:
 
-* Initializing [local state](/docs/state-and-lifecycle.html) by assigning an object to `this.state`.
-* Binding [event handler](/docs/handling-events.html) methods to an instance.
-
-You **should not call `setState()`** in the `constructor()`. Instead, if your component needs to use local state, **assign the initial state to `this.state`** directly in the constructor:
+* Inicializálni a [helyi állapotot](/docs/state-and-lifecycle.html) egy objektum `this.state`-hez rendelésével.
+* Hozzákötni [eseménykezelő](/docs/handling-events.html) metódusokat egy komponenspéldányhoz.
+  
+**Nem szabad a `setState()`-et** meghívni a `constructor()`-ban. Ehelyett, ha a komponensednek helyi állapotot kell használnia, **rendeld hozzá a kezdő állapotot a `this.state`-hez** közvetlenül a konstruktorban:
 
 ```js
 constructor(props) {
   super(props);
-  // Don't call this.setState() here!
+  // Ne hívd meg a this.setState()-et itt!
   this.state = { counter: 0 };
   this.handleClick = this.handleClick.bind(this);
 }
 ```
 
-Constructor is the only place where you should assign `this.state` directly. In all other methods, you need to use `this.setState()` instead.
+A konstruktor az egyetlen hely, ahol a `this.state`-hez közvetlenül hozzárendelhetsz egy értéket. Mindenhol máshol a `this.setState()`-et használd ehelyett.
 
-Avoid introducing any side-effects or subscriptions in the constructor. For those use cases, use `componentDidMount()` instead.
+Kerüld el a mellékhatások bevezetését vagy a feliratkozásokat a konstruktorban. Ezekre az esetekre inkább a `componentDidMount()`-ot használd.
 
->Note
+>Megjegyzés
 >
->**Avoid copying props into state! This is a common mistake:**
+>**Kerüld el a propok helyi állapotba való másolását! Ez egy gyakori hiba:**
 >
 >```js
 >constructor(props) {
 >  super(props);
->  // Don't do this!
+>  // Ezt ne csináld!
 >  this.state = { color: props.color };
 >}
 >```
 >
->The problem is that it's both unnecessary (you can use `this.props.color` directly instead), and creates bugs (updates to the `color` prop won't be reflected in the state).
+>A probléma az, hogy ez mind felesleges (a `this.props.color`-t közvetlenül is tudod használni ehelyett), mind pedig hibákat generál (a `color` prop frissülése nem fog megjelenni a helyi állapotban).
```

Comment:
```suggestion
>A probléma az, hogy ez mind felesleges (a `this.props.color`-t közvetlenül is tudod használni ehelyett), mint pedig hibákat generál (a `color` prop frissülése nem fog megjelenni a helyi állapotban).
```

Reply:
Sorry it's not a typo, it was meant to be 'mind ..., mind' as in 'both ..., and'
Feel free to suggest a better way of saying this :)